src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -200,6 +201,11 @@
         ],
         "Bridged Device Basic Information": ["ProductAppearance"],
         "Chime": ["ActiveChimeID", "Enabled"],
+        "Closure Control": [
+            "OverallState",
+            "OverallTarget",
+            "FeatureMap"
```

Comment:
FeatureMap should not be added here, because that's implementation-dependent.  Should be added in the implementation PR, when it's possible to check that the implementation handles it correctly.

Reply:
Updated.